public/gh-reroute/404.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
```

Comment:
![image](https://github.com/allen-cell-animated/website-3d-cell-viewer/assets/30200665/f796f2eb-2ca3-4764-bf94-7d5a8b8208de)

^ here's what this page looks like! It should only briefly flash across the screen, so I tried to make sure it wouldn't be too obtrusive. I unfortunately don't think I can include a link back to the root of the viewer without some more complex configuration, so I settled on the github repo for now.

Reply:
This may show for a little longer than it used to... I might be able to make it faster by not including the Open Sans font override, but it looks more consistent this way.

Reply:
Showing this page only happens in gh-pages builds, right?  Will end users of staging or production see this?

Reply:
Also, was this made worse because of refactoring the code?  Was it less noticeable before?

Reply:
This will only be shown on GitHub pages, yes. I think it's worse because it's loading the separate code bundle in addition to just the plain HTML, and there's some boilerplate that makes the overall filesize larger.